src/main/java/org/opentripplanner/routing/alertpatch/AlertPatch.java

```diff
@@ -84,17 +89,17 @@ public void setId(String id) {
         this.id = id;
     }
 
-    public void apply(Graph graph) {
-        Agency agency = null;
+    public void applyOrRemove(Graph graph, Consumer<Edge> applyOrRemoveFn) {
+        Agency alertAgency = null;
```

Comment:
This variable name change feels a little weird only because the other entities (route and trip) weren't updated similarly. Is there a reason for that?

Reply:
I forget, so I changed it back in d074833db78929d4776ad0ac37724965657cead7